cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/AbstractCruiseControlResponse.java

```diff
@@ -28,6 +29,8 @@ public void writeSuccessResponse(CruiseControlParameters parameters, HttpServlet
     OutputStream out = response.getOutputStream();
     boolean json = parameters.json();
     setResponseCode(response, SC_OK, json, null);
+    response.addHeader("Cruise-Control-Version", KafkaCruiseControl.cruiseControlVersion());
+    response.addHeader("Cruise-Control-Commit_Id", KafkaCruiseControl.cruiseControlCommitId());
```

Comment:
I think you also need to add this header logic to [writeResponseToOutputStream](https://github.com/linkedin/cruise-control/blob/c3d3ece6debc9e11c0207c09d0b32d05caf0141c/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ResponseUtils.java#L59).
It is used to return in-progress and error response.